engine/lite/kafka/api/src/main/scala/pl/touk/nussknacker/engine/lite/kafka/api/LiteKafkaSource.scala

```diff
@@ -9,4 +10,10 @@ trait LiteKafkaSource extends BaseLiteSource[ConsumerRecord[Array[Byte], Array[B
 
   def topics: List[String]
 
+  def transform(record: ConsumerRecord[Array[Byte], Array[Byte]]): Context = {
+
+    Context(contextIdGenerator.nextContextId())
```

Comment:
why do you add this variable this explicitly and it test impl `super.transform(record)` implicitly ?